onnxruntime/python/tools/transformers/convert_to_onnx.py

```diff
@@ -159,6 +159,14 @@ def parse_arguments(argv=None):
     fp16_option_group = parser.add_argument_group(
         "float to float16 conversion parameters that works when \"--precision fp16\" is specified")
 
+    fp16_option_group.add_argument(
+        '-a',
+        '--auto_mixed_precision',
+        required=False,
+        action='store_true',
+        help='Convert to mixed precision automatically. Other float16 conversion parameters will be ignored.')
```

Comment:
Is there a reason to allow other parameters for float16? Any fallback mechanism uses those parameters?

Reply:
The fallback mechanism: when use opt to auto, we will generate all parameters automatically. Otherwise, use has to specify other parameters manually.